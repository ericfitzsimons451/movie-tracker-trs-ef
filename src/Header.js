import React from 'react'

const Header = () => {
    return (
        <div>
            <h1>Luke Sperry's Movie Trrrrrrrrrracker</h1>
        </div>
    )
}

export default Header