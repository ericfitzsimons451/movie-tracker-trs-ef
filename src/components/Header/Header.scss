.logo {
    color: #db0000;
    width: 1fr;
    display: flex;
    align-items: center;
    font-size: 1.5rem;
    font-family: 'Oswald', sans-serif;
}

