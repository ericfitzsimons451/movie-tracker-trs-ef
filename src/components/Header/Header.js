import React from "react";
import "./Header.scss";
import "./Header";

const Header = () => {
  return (
    <div className="logo">
      <h1>MOVIE TRACKER</h1>
    </div>
  );
};

export default Header;
