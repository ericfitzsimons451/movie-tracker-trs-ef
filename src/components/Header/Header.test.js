import Header from './Header'
import React from 'react'
import { shallow } from 'enzyme'

describe('Header', () => {
    let wrapper;
    it('matches the snapshot', () => {
        wrapper = shallow(<Header />)
        expect(wrapper).toMatchSnapshot()
    })
})