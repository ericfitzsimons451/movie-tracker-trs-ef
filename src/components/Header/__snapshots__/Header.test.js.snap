// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches the snapshot 1`] = `
<div
  className="logo"
>
  <h1>
    MOVIE TRACKER
  </h1>
</div>
`;
